### ISSUE TITLE
Fix "PastAssignments" page on the student end.

### ISSUE THREAD

**Issue by GhTmp1A** [sitechtimes/regents-prep-app-frontend#190]
When a student clicks "past assignments" on the classpreview component from their own dashboard, the page they enter is incomplete. When they click on this, they should be brought to the class page, where all of the assignments are displayed. The dropdown menu must be brought to "Missing" assignments. 
(Currently On Hold and waiting on #172's completion)
